EntAppProject/src/main/resources/templates/JBUserProfile.html

```diff
@@ -23,12 +23,13 @@ <h2><a href="/JBWelcomeScreen">
 <form id="bio" onsubmit="return false;">
 Bio:
 <br>
-<input style=position:left; id="userInput">
+<textarea style="width: 250px; height:100px;" id="userInput"placeholder="User description here"></textarea>
+
 
 <br>
 Skills:
 <br>
-<input style=position:left; type="text" id="userSkills">
+<textarea style="width: 250px; height:100px;" id="userSkills"placeholder="Skill description here"></textarea>

```

Comment:
Changed the textboxs to a text areas for the user bio and user skill as this allowed more room to input text.

Reply:
Ditto comment above... make inline styles linked styles instead.